### ISSUE TITLE
fetch ar/state_health_units

### ISSUE THREAD

**Issue by Vaccinebot** [CAVaccineInventory/vaccine-feed-ingest#693]
[![learn our pipeline: fetch](https://img.shields.io/static/v1?label=learn%20our%20pipeline&message=fetch&style=social)](https://github.com/CAVaccineInventory/vaccine-feed-ingest/wiki/Runner-pipeline-stages#fetch)

Fetch data from: https://www.healthy.arkansas.gov/local-health-units

Put your script in a file named: `ar/state_health_units/fetch.py` (or `fetch.sh`, etc.)

Store it (without processing) in a new file created in the directory passed as the first argument (`sys.argv[1]`).

Check the wiki to learn more about the purpose of the fetch stage and how to get set up for development!

### Tips

1. While working on your code, run it at any point:
    ```sh
    poetry run vaccine-feed-ingest fetch ar/state_health_units
    ```

**Comment by shashank025** [comment]
I'll take this.

**Comment by shashank025** [comment]
Reopening to reflect the fact that the prior fetch script wasn't cutting it. Should be closed once #720 is submitted.
